ts/Extensions/ExportData/ExportDataDefaults.ts

```diff
@@ -265,7 +265,9 @@ const lang: LangOptions = {
      * @since 8.2.0
      * @requires modules/export-data
      */
-    hideData: 'Hide data table'
+    hideData: 'Hide data table',
+
+    exportInProgress: 'Exporting...'
```

Comment:
I think, that kind of labels (`Exporting...`) should be in `lang` object, same as e.g. [loading label](https://api.highcharts.com/highcharts/lang.loading).

Reply:
I think it kind of is? I see that `hideData` ends up in the same API location and it is defined in the same place 🤔

Reply:
Ahh sorry! My bad!